yt/data_objects/static_output.py

```diff
@@ -822,17 +838,20 @@ def _get_field_info(self, ftype, fname=None):
         guessing_type = ftype == "unknown"
         if guessing_type:
             ftype = self._last_freq[0] or ftype
+            ambiguous = fname in self.field_info._ambiguous_field_names
```

Comment:
So that it's more obvious that it's a bool
```suggestion
            is_ambiguous = fname in self.field_info._ambiguous_field_names
```